src/main/java/baseball/BaseballGame.java

```diff
@@ -0,0 +1,83 @@
+package baseball;
+
+import camp.nextstep.edu.missionutils.Console;
+import java.util.List;
+
+public class BaseballGame {
+    private boolean runningGame = true;
+    private static final int MAX_NUM_LENGTH = 3;
+
+    public void getGame() {
+        GetNumbers gn = new GetNumbers();
+        List<Integer> computerNum = gn.getRandomNumber();
+        getGameProcess(computerNum);
+    }
+
+    public void getGameProcess(List<Integer> computer) {
+        while (runningGame) {
+            getGameResult(computer);
+        }
+    }
+
+    public void getGameResult(List<Integer> computer) {
+        GetNumbers gn = new GetNumbers();
+        List<Integer> user = gn.getUserInput();
+        int ballCount = 0;
+        int strikeCount = 0;
+
+        for (int i = 0; i < MAX_NUM_LENGTH; i++) {
+            if (computer.contains(user.get(i)) && !computer.get(i).equals(user.get(i))) {
+                ballCount++;
+            }
+            if (computer.get(i).equals(user.get(i))) {
+                strikeCount++;
+            }
+        }
+
+        checkCountResult(ballCount, strikeCount);
+    }
+
+    public int selectExit() {
+        System.out.println("게임을 새로 시작하려면 1, 종료하려면 2를 입력하세요.");
+        int choice = Integer.parseInt(Console.readLine());
+        int selectNum = 1;
+        if (choice != 1) {
+            selectNum = -1;
+        }
+        return selectNum;
+    }
```

Comment:
```suggestion
    public boolean selectExit() {
        /*
        ...
        */
        int isGameTobeContinued = true;
        if (choice != 1) {
            selectNum = false;
        }
        return selectNum;
    }
```
여기서 1이나 -1은 이 코드를 처음 본 개발자들은 그 의미를 알기 어렵지 않을까요? 이처럼 하드 코딩된 숫자나 문자열을 매직 넘버라고 하는데요. 문자열 하드코딩이나 매직 넘버는 위에서 언급했듯이 지양하는 것이 좋습니다.

여기서는 게임을 계속 할 것이냐 말 것이냐 두 가지이므로 int 값보다는 의미있는 이름을 가진 boolean 변수를 선언하는 것이 어떨까요?

https://velog.io/@hwisaac/%EB%A7%A4%EC%A7%81%EB%84%98%EB%B2%84